packages/browser-wallet-api/src/wallet-api.ts

```diff
@@ -57,9 +59,16 @@ class WalletApi implements IWalletApi {
     /**
      * Sends a sign request to the Concordium Wallet and awaits the users action
      */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    public signMessage(): Promise<any> {
-        return this.sendMessage(MessageType.SignMessage, {});
+    public async signMessage(message: string): Promise<AccountTransactionSignature> {
```

Comment:
`IWalletAPI` interface needs to be changed to match this.

Reply:
:weary: good catch.